source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -1024,23 +1022,23 @@ bool COpenGLDriver::genericDriverInit()
 	// Reset The Current Viewport
 	glViewport(0, 0, Params.WindowSize.Width, Params.WindowSize.Height);
 
-	glEnable(GL_FRAMEBUFFER_SRGB);
-    glDisable(GL_DITHER);
-    glDisable(GL_MULTISAMPLE);
-    glEnable(GL_TEXTURE_CUBE_MAP_SEAMLESS);
-    extGlClipControl(GL_UPPER_LEFT, GL_ZERO_TO_ONE);
-	glClearDepth(0.0);
-	glDepthFunc(GL_GEQUAL);
-	glDepthRange(1.0,0.0);
-	glFrontFace(GL_CCW);
-
 	// adjust flat coloring scheme to DirectX version
 	///extGlProvokingVertex(GL_FIRST_VERTEX_CONVENTION_EXT);
 
 	// We need to reset once more at the beginning of the first rendering.
 	// This fixes problems with intermediate changes to the material during texture load.
-    //TODO hm? should i worry about that?
-	ResetRenderStates = true;
+    SAuxContext* found = getThreadContext_helper(false);
+    found->nextState.rasterParams.framebufferSRGB = 1;
```

Comment:
frambufferSRGB is always true, and non debatable (shouldn't even be in the state)

Reply:
resolved